### PR TITLE
Fixes cryo indicating people with 0 health as DEAD and also now indicates how far in crit.

### PR DESCRIPTION
**Pull request by invoid9**
Fixes an issue I've never heard anyone complain about. Tested, shows people at 0 health as being in crit, which doesn't seem to be an issue to me.

### CODE REVIEW

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -380,7 +380,7 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 								overlays += cryo_health_indicator["average"]
 							if(25 to 50)
 								overlays += cryo_health_indicator["bad"]
-							if(1 to 25)
+							if(0 to 25)
```
Comment (ghost) [inline code/game/machinery/cryo.dm]: I'm surprised this doesn't go off health percentage, but then again health percentage doesn't make sense in itself.

Comment (invoid9) [reply code/game/machinery/cryo.dm]: I don't know how any of this works, but this seemed to fix it so :shrug_emoji:

Comment (Intigracy) [reply code/game/machinery/cryo.dm]: `switch((occupant.health / occupant.maxHealth) * 100) // Get a ratio of health to work with`
It is a percentage.

**Comment by Kammerjunk** [comment]
I think no-one complained because it's been here forever and people just got used to it. Nice to get rid of.

File: code/game/machinery/cryo.dm (hunk 1 above)
Comment (clusterfack) [inline code/game/machinery/cryo.dm]: The problem here is, health ranges from 100% to -100%, with 0% being the critical health point. This percentage switch statement doesn't take this into account.

**Comment by invoid9** [comment]
Gonna add a new commit in a second unless I've fucked something up. It DOES indicate crit the same way it indicates health above crit but in reverse. The overlay for when they're dead is just a full red bar that does not flash, to distinguish between near death and death.

This is probably not the most elegant way to do this, but I've tested it and it seems to work.

http://imgur.com/a/Xf6FV
http://imgur.com/a/APPmu

**Pull request merged**